src/wrappers/wrapper.js

```diff
@@ -109,7 +109,11 @@ export default class Wrapper implements BaseWrapper {
       targetClass = this.vm.$style[targetClass]
     }
 
-    return !!(this.element && this.element.classList.contains(targetClass))
+    const containsAllClasses = targetClass
```

Comment:
I think containsAllClasses should be a function, otherwise it will error if this.element is undefined, whereas at the moment the method returns false

Reply:
Won't it still throw an error if this.element is undefined (since `this.element.className.contains` will throw the "Cannot find .contains of null" type error, even if containsAllClasses is a function?